config/indexer.go

```diff
@@ -24,7 +26,10 @@ type Indexer struct {
 	LayerScanConcurrency int `yaml:"layer_scan_concurrency" json:"layer_scan_concurrency"`
 	// Rate limits the number if index report creation requests.
 	//
-	// Any value below 1 is considered unlimited.
+	// Setting this to 0 will attempt to auto-size this value. Setting a
```

Comment:
Do you think we should set this value to -1 in the config created by the operator or adopt a new default behaviour? I'm in favor of the 2nd but we might need to flag it to support to avoid cases being created.

Reply:
I think we want the default to be auto-sized, but let's ask the operator team on their thoughts.

Reply:
@jonathankingfc said on IM that "auto" is probably what we want.